packages/components/src/color-picker/styles.ts

```diff
@@ -30,7 +30,7 @@ export const SelectControl = styled( InnerSelectControl )`
 	margin-left: ${ space( -2 ) };
 	width: 5em;
 	${ BackdropUI } {
-		display: none;
+		display: flex;
```

Comment:
I used flex because other elements like Input box for color value and copy button both uses flex. Otherwise block was also working 😄.  Was one of the two failed checks because of flex. Failure of Changelog diff check I understood. But the reason for another failed check is  not clear to me.

Reply:
I didn't see it, but if you saw another failure it may have been a flaky end-to-end test. Everything looks good now, so I'll merge. :)